src/_data/toc/software-update-guide.yml

```diff
@@ -1,16 +1,16 @@
 label: Software Update Guide
 pages:
 
-    - label: Upgrade the Magento application and modules
+    - label: Magento upgrade overview
       url: /comp-mgr/bk-compman-upgrade-guide.html
 
     - label: Prerequisites
       url: /comp-mgr/prereq/prereq_compman.html
       children:
 
         - label: Check the catalog search engine
-          url: /comp-mgr/prereq/prereq-elasticsearch.html
-          exclude_versions: ["2.3"] 
+          url: /comp-mgr/prereq/prereq-elasticsearch.html	
```

Comment:
It looks like there's a tab after the URL

Reply:
Good eye!